docs/design-reviews/decision-log/examples/memory/Architecture/Data-Model.md

```diff
@@ -0,0 +1,267 @@
+# Data Model
+
+## Table of Contents
+
+- [Graph vertices and edges](#Graph-vertices-and-Edges)
+- [Graph Properties](#Graph-Properties)
+- [Vertex Descriptions](#Vertex-Descriptions)
+- [Full Role JSON Example](#Full-Role-JSON-Example)
+
+## Graph Model
+
+## Graph Vertices and Edges
+
+The set of vertices (entities) and edges (relationships) of the graph model
+
+| Vertex (Source) | Edge Type        | Relationship Type | Vertex (Target) | Notes                                                                                                | Required |
```

Comment:
```suggestion
| Vertex (Source) | Edge Type       | Relationship Type | Vertex (Target) | Notes                                                                                                | Required |
```

Reply:
making a space change to trigger a rerun of the checks -- for some reason I can't trigger the checks automagically for this PR